stargazer/stargazer.py

```diff
@@ -111,7 +112,8 @@ def extract_data(self):
         covs = []
         for md in self.model_data:
             covs = covs + list(md['cov_names'])
-        self.cov_names = sorted(set(covs))
+        # deduplicate cov_names list from multiple models with same dep-vars (but keep order seen)
+        self.cov_names = list(dict.fromkeys(covs))
```

Comment:
Can you clarify what is the problem?

Reply:
This was a mistake on my part, it's a local change on my part that I didn't mean to include it in the t/p stat change. This code does the same thing as sorted(set()) (removing duplicates) but doesn't sort the covariates, so the output order will be the same as the input order. I'll remove this change.

Reply:
Thanks